src/katatraining/fibonacci/Fibonacci.java

```diff
@@ -0,0 +1,13 @@
+package katatraining.fibonacci;
+
+public class Fibonacci {
+    public static long fib(int n) {
+        int[] x = new int[n + 1];
+        x[0] = 0;
+        x[1] = 1;
```

Comment:
You set zeroth x[0] = 0. Does it mean that someone could use your method to find element with index zero? If it's so then you will get and exception. 

Just write another one test in FibonacciTest class for that case and you'll see where is the problem.

Fibonacci.fib(0)